pwndbg/dbg/__init__.py

```diff
@@ -142,6 +142,13 @@ class BreakpointLocation:
     def __init__(self, address: int):
         self.address = address
 
+    def __eq__(self, other: object) -> bool:
```

Comment:
Do we really need this class? It is a pure int under the hood...

Could maybe be just a type alias for mypy typing?

Reply:
It mimics lldb's (and gdb 13.1+) concept I guess. The real classes have more fields. I'm fine to convert it to a pure type alias for now if you want to.